src/metricq-history.js

```diff
@@ -59,10 +59,12 @@ class MetricQHistoric {
     this.url = url
   }
 
-  search (target) {
+  search (target, metadata = false, limit = undefined) {
     return new Promise((resolve, reject) => {
       axios.post(`${this.url}/search`, {
-        target: target
+        target: target,
+        metadata,
+        limit
```

Comment:
```suggestion
        metadata: metadata,
        limit: limit
```

Shouldn't it be like that ^

Reply:
These are shorthand property names (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#new_notations_in_ecmascript_2015). Webpack supports them, so the packed js should work fine even without browser support.

Reply:
Fine, but keep it consistent, please.